platform/net/ubuntu_net_manager.go

```diff
@@ -102,6 +102,11 @@ func (net UbuntuNetManager) SetupNetworking(networks boshsettings.Networks, errC
 		return net.writeResolvConf(networks)
 	}
 
+	// Specific for softlayer networking
```

Comment:
Cool, great comment. Though let's be consistent while not adding this everywhere. Perhaps first time SL-specific code shows up. Thx